background.ts

```diff
@@ -519,37 +518,19 @@ Electron.ipcMain.handle('service-forward', async(event, service, state) => {
 });
 
 Electron.ipcMain.on('k8s-integrations', async(event) => {
-  if (k8smanager instanceof WSLBackend) {
-    event.reply('k8s-integrations', await k8smanager?.listIntegrations());
-  }
+  // listIntegrations triggers integration-update, no need to deal with results.
+  await integrationManager.listIntegrations();
```

Comment:
Why not just do `mainEvents.emit('integration-update', await integrationManager.listIntegrations())`? It seems confusing to me to have `integrationManager.listIntegrations` have the side effect of emitting an event - the name has nothing to do with the event. Plus it makes the reader step through another layer of indirection.